ui/components/multichain/notification-list-item/index.scss

```diff
@@ -12,11 +12,15 @@
 }
 
 .notification-list-item--unread {
-  background-color: rgba(3, 109, 185, 0.05); // #0376C9 at 5% opacity
+  [data-theme='light'] & {
+    --item-background: rgba(3, 118, 201, 0.05);
+  }
 
-  &:hover {
-    background-color: rgba(3, 109, 185, 0.1); // #0376C9 at 10% opacity
+  [data-theme='dark'] & {
+    --item-background: rgba(3, 118, 201, 0.1);
   }
```

Comment:
Unread styles to match our figma - the deviate from out DS colors (I think they are our DS colors by with more opacity)